ui/src/models/workflows.ts

```diff
@@ -776,6 +776,10 @@ export interface Condition {
 export type ConditionType = 'Completed' | 'SpecWarning' | 'MetricsError' | 'SubmissionError' | 'SpecError' | 'ArtifactGCError';
 export type ConditionStatus = 'True' | 'False' | 'Unknown';
 
+export interface WorkflowsPaginationOptions {
+    wfContinue: string;
+    archivedContinue: string;
+}
```

Comment:
```suggestion
}

```

missing a new line as is convention in this file between all the types